vscode-dotnet-runtime-library/package.json

```diff
@@ -25,6 +25,7 @@
 	},
 	"devDependencies": {
 		"@types/chai": "4.2.22",
+		"@types/lodash": "^4.17.13",
```

Comment:
I'm absolutely not against using new packages and things, but does C# DevKit have any restrictions for new packages? Just wondering from a security standpoint.

Reply:
Thanks for taking a look :) There shouldn't be any restrictions besides license / legal / security, which are handled internally by our tooling.